engine/moduleManager/modules/rendering/ImGuiRenderingModule.hpp

```diff
@@ -0,0 +1,35 @@
+//
+// Created by 11896 on 06/12/2024.
+//
+
+#ifndef BLOCKYENGINE_ENGINE_MODULEMANAGER_MODULES_RENDERING_IMGUIRENDERINGMODULE_HPP_
+#define BLOCKYENGINE_ENGINE_MODULEMANAGER_MODULES_RENDERING_IMGUIRENDERINGMODULE_HPP_
+
+#include "moduleManager/ModuleWrapper.hpp"
+#include <SDL.h>
```

Comment:
is hier geheel SDL en imgui nodig of kunnen subsecties gebruikt worden in de header. let erop dat je niet onnodig veel include in een header, enkel wat je nodig hebt